tex/generic/pgf/utilities/pgfutil-common.tex

```diff
@@ -910,5 +919,6 @@
     \endgroup
 }%
 \input pgfutil-common-lists.tex
+\input pgfutil-common-expl.tex
```

Comment:
This should be inlined into `pgfutil-common.tex` directly. Loading files in TeX is slow. @josephwright promised to develop some kind of luadocstrip for l3build to merge files during build. Once that arrives we can split it again.

Reply:
Made inlined in commit https://github.com/pgf-tikz/pgf/pull/1124/commits/190342a19826f6ab485db81d28a8d624c90be32c

Reply:
Since `pgfutil-<format>.def` is loaded _after_ `pgfutil-common.tex`, all wrapped primitives [1] are undefined in `pgfutil-common.tex`. We have to pay attention to not actually expand any of them in defining `pgfexpl` macros. Not sure if this is achievable in the long term.

[1] Currently there're `\pgfutil@protected` and `\pgfutil@unexpanded`. `\pgfutil@expanded` should be added.